src/LinkMe/LinkMe.Api/LinkMe.Api.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <InvariantGlobalization>false</InvariantGlobalization>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
there are couple of CSPROJ files we need to clean up
- dotNET offers transitive dependencies, no need to include `<PackageReference>` in all projects. For example, EF can be in Infrastructure only, while Api project to reference ApplicationServices only.
- since I see `Infra.Sqlite` and `Infra.SqlServer`, we need a DependencyInjection project to map the Registry (StructureMap language) in one place. This will help with moving `Infra.X` in all CSPROJ apps.
- (optional) Domain has access to EF even though having Clean Architecture means our domain is not bound by any ORM framework
- (idea) We can make `LinkMe.Infrastructure.csproj` be our DependencyInjection layer, where it contains all other projects. However, the idea with `Infra.Sqlite` makes this a cyclic dependency.

I can add Infrastructure tests, so that Quality gates fail in case they are re-added.

Reply:
Need to consult with @dragandimkovikj on why `LinkMe.Api` is the bootstrapper and has all Infrastructure dependencies. It should only have `LinkMe.Infrastructure` 

Idea is to reach the following setup
https://github.com/amantinband/clean-architecture/tree/main

while using the following tests to assert the architecture
https://youtu.be/_D6Kai4RdGY

Maybe this is a starting architecture, not the final one.